ui/pages/create-account/import-account/index.js

```diff
@@ -1,79 +1,160 @@
+      );
     }
+
+    return '';
   }
 
-  render() {
-    const menuItems = this.getMenuItemTexts();
-    const { type } = this.state;
-    const { t } = this.context;
-
-    return (
-      <>
-        <div className="page-container__header">
-          <div className="page-container__title">{t('importAccount')}</div>
-          <div className="page-container__subtitle">
-            {t('importAccountMsg')}
-            <span
-              className="new-account-info-link"
-              onClick={() => {
-                global.platform.openTab({
-                  url: ZENDESK_URLS.IMPORTED_ACCOUNTS,
-                });
-              }}
-            >
-              {t('here')}
-            </span>
-          </div>
-        </div>
-        <div className="new-account-import-form">
-          <div className="new-account-import-form__select-section">
-            <div className="new-account-import-form__select-label">
-              {t('selectType')}
-            </div>
-            <Dropdown
-              className="new-account-import-form__select"
-              options={menuItems.map((text) => ({ value: text }))}
-              selectedOption={type || menuItems[0]}
-              onChange={(value) => {
-                this.setState({ type: value });
-              }}
-            />
-          </div>
-          {this.renderImportView()}
-        </div>
-      </>
-    );
+  /**
+   * @param {string} message - an Error/Warning message caught in importAccount()
+   * This function receives a message that is a string like:
+   * `t('importAccountErrorNotHexadecimal')`
+   * `t('importAccountErrorIsSRP')`
+   * `t('importAccountErrorNotAValidPrivateKey')`
+   * and feeds it through useI18nContext
+   */
+  function translateWarning(message) {
+    if (message && !message.startsWith('t(')) {
+      // This is just a normal error message
+      dispatch(actions.displayWarning(message));
+    } else {
+      // This is an error message in a form like
+      // `t('importAccountErrorNotHexadecimal')`
+      // so slice off the first 3 chars and last 2 chars, and feed to i18n
+      dispatch(actions.displayWarning(t(message.slice(3, -2))));
+    }
+  }
+
+  function PrivateKeyOrJson() {
+    switch (type) {
+      case menuItems[0]:
+        return <PrivateKeyImportView importAccountFunc={importAccount} />;
+      case menuItems[1]:
+      default:
+        return <JsonImportView importAccountFunc={importAccount} />;
+    }
   }
+
+  return (
+    <>
+      <Box
+        padding={4}
+        style={{ borderWidth: '0px 0px 1px 0px' }} // There is no way to do just a bottom border in the Design System
+        borderColor={BorderColor.borderDefault}
+      >
+        <Text variant={TextVariant.headingLg}>{t('importAccount')}</Text>
+        <Text variant={TextVariant.bodySm}>
+          {t('importAccountMsg')}{' '}
+          <ButtonLink size={Size.inherit} onClick={moreInfoLink}>
+            {t('here')}
+          </ButtonLink>
```

Comment:
In the current Ui, we have an underline for this text. Are we removing it in this design?

### Before 
![n bmh](https://user-images.githubusercontent.com/39872794/220892888-4e649170-215b-4a9f-b674-ffe2999eb0c6.jpg)

### After
![ bb](https://user-images.githubusercontent.com/39872794/220892854-51d78701-4fdb-43b8-8f44-74647f8c2240.jpg)

Reply:
Are we updating the `Text` colors as well @HowardBraham ?

Reply:
@NidhiKJha the removal of the underline and the change of the Text colors are not exactly intentional, but just a consequence of going with the defaults for the Design System.  We may want to discuss if we want to change these defaults, if we want to let it be, or if we want to customize this page.